api/organisations/migrations/0053_create_api_billing.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.25 on 2024-04-08 15:07
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('organisations', '0052_create_hubspot_organisation'),
+    ]
+
+    operations = [
+        migrations.RenameModel(
+            old_name='OranisationAPIUsageNotification',
+            new_name='OrganisationAPIUsageNotification',
```

Comment:
I think that since nothing is using the table right now we should just rename the table. Nothing is going to break and who wants to deal with a legacy typo for decades?